sdk/cognitivelanguage/ai-language-text-authoring/samples/WebOfScience/001.txt

```diff
@@ -0,0 +1 @@
+[Purpose] This study verified that the smoothness of reaching movements is able to quantitatively evaluate the effects of two-and three-dimensional images on movement in healthy people. In addition, clinical data of cerebrovascular accident patients were also analyzed by the same method. [Subjects] Ten healthy adult volunteers and two male patients with previous cerebrovascular accidents participated. [Methods] The subjects were tasked with reaching for objects shown on a display. The target and virtual limb, rendered with computer graphics, were shown on the display. Movements of the virtual limb were synchronized with those of the subject. Healthy subjects reached for targets with their dominant arm, and cerebrovascular accident patients used their paretic arm. A polarized display and polarized glasses were used when the subjects were shown three-dimensional images. In the present study, jerk cost was used to quantify the smoothness of movement. [Results] Six of the 10 healthy subjects had significantly smoother reaching movements when viewing the three-dimensional images. The two cerebrovascular accident patients tended to have smoother movements in response to the three-dimensional images. [Conclusion] Analysis of the smoothness of movement was able to detect the influence of the depth cue in vision on movement quantitatively for the healthy subjects and cerebrovascular accident patients.
```

Comment:
Do we need these files in the samples directory? We don't have to worry about samples for this library at the moment.

Reply:
No, we do not need these files in this directory at all. This entire samples folder in the authoring library will be deleted since it is all integrated in the custom tests in the language library already